safe/definitions/fields.py

```diff
@@ -2505,6 +2505,109 @@
     'units': currencies,
 }
 
+# # # # # # # # # #
+# Direction and Distance Field
+# # # # # # # # # #
+
+# Count for each exposure type
+distance_field = {
+    'key': 'distance_field',
+    'name': tr('Distance'),
+    'field_name': 'distance',
+    'header_name': tr('Distance'),
+    'type': QVariant.Double,
+    'length': default_field_length,
+    'precision': default_field_precision,
+    'absolute': True,
+    'description': tr(
+        'The distance value between place and hazard point.'
+    ),
+    'help_text': tr(
+        'Distance value between place feature to the epicenter of the hazard. '
+        'The distance is calculated using WGS84 as ellipsoid model. The unit of'
+        'the distance is in meter.'),
+    'citations': [
+        {
+            'text': None,
+            'link': None
+        }
+    ],
+    # Null value can be replaced by default or not
+    'replace_null': False
+}
+
+bearing_from_field = {
+    'key': 'bearing_from_field',
+    'name': tr('Bearing Angle'),
+    'field_name': 'bearing_fr',
+    'header_name': tr('Bearing'),
+    'type': QVariant.Double,
+    'length': default_field_length,
+    'precision': default_field_precision,
+    'absolute': False,
+    'description': tr(
+        'An azimuth angle at a place to a hazard point.'),
+    'help_text': tr(
+        'A bearing angle is an angle measured to a point as observed in current'
+        ' location using north as a reference direction. In this case, "bearing'
```

Comment:
Always put `space` character in the end of string, not in the begining of string. I see there are some more.

Reply:
OK, I will update it.